FirebaseRemoteConfig/Tests/SwiftAPI/APITests.swift

```diff
@@ -18,10 +18,24 @@ import FirebaseCore
 import XCTest
 
 class APITests: APITestBase {
+  var console: RemoteConfigConsole!
+
   override func setUp() {
     super.setUp()
     if APITests.useFakeConfig {
       fakeConsole.config = ["Key1": "Value1"]
+    } else {
+      console = RemoteConfigConsole()
+      console.updateRemoteConfigValue("Obi-Wan", for: "Jedi")
+      console.removeRemoteConfigValue(for: "Sith_Lord")
+    }
+  }
```

Comment:
@paulb777 Since we have a few cases here (FakeConsole, RemoteConfigConsoleAPI, and regular Remote Config API calls), I was thinking I need to either organize it better so only the tests affiliated with each case run or just separate this file into separate ones based on how we are testing (fake, real, regular API). Happy to do either but I'd appreciate some input

Reply:
Maybe it's ok to drop the regular Remote Config API test calls? The RemoteConfigConsoleAPI is a superset that is still talking to a real backend. Right?

Reply:
Yup, that is correct.